cmake/developer_package/download/download_and_extract.cmake

```diff
@@ -5,9 +5,13 @@
 include(download/extract)
 include(download/download_and_check)
 
-function (GetNameAndUrlToDownload name url archive_name_unified archive_name_win archive_name_lin archive_name_mac archive_name_android)
+function (GetNameAndUrlToDownload name url archive_name_unified archive_name_win archive_name_lin archive_name_mac archive_name_android USE_NEW_LOCATION)
```

Comment:
it is strange that in some cmake files there are used lower case only and other cmake-s upper case only names.

maybe it would be good to be have names consistent with current file (if we modify existing one).